### ISSUE TITLE
Underline errors and warnings then "apply"

### ISSUE THREAD

**Issue by walterty** [spyder-ide/spyder#19227]
### What steps will reproduce the problem?

Go to Tools>Preferences>Linting and select or deselect "Underline errors and warnings". Then apply the setting.

The issue reporter immediately launched; however, it looks like the settings are applied properly. I see underlining with errors (or not when the setting is deselected).

### Traceback
```python-traceback
Traceback (most recent call last):
  File "C:\Users\walte\anaconda3\lib\site-packages\spyder\plugins\editor\plugin.py", line 419, in _rpc_call
    meth(*args, **kwargs)
  File "C:\Users\walte\anaconda3\lib\site-packages\spyder\plugins\editor\plugin.py", line 1292, in update_source_menu
    state = self.get_option(opt, section=section)
  File "C:\Users\walte\anaconda3\lib\site-packages\spyder\api\plugins\old_api.py", line 111, in get_option
    return super(BasePlugin, self)._get_option(option, default,
  File "C:\Users\walte\anaconda3\lib\site-packages\spyder\plugins\base.py", line 71, in _get_option
    return CONF.get(section, option, default)
  File "C:\Users\walte\anaconda3\lib\site-packages\spyder\config\manager.py", line 459, in get
    raise cp.NoOptionError(option, section)
configparser.NoOptionError: No option ('editor', 'underline_errors') in section: 'editor'
```

## Versions

* Spyder version: 5.2.2 
* Python version: 3.9.12
* Qt version: 5.9.7
* PyQt5 version: 5.9.2
* Operating System: Windows 10

### Dependencies

```
# Mandatory:
atomicwrites >=1.2.0          :  1.4.0 (OK)
chardet >=2.0.0               :  4.0.0 (OK)
cloudpickle >=0.5.0           :  2.0.0 (OK)
cookiecutter >=1.6.0          :  1.7.3 (OK)
diff_match_patch >=20181111   :  20200713 (OK)
intervaltree >=3.0.2          :  3.1.0 (OK)
IPython >=7.6.0;<8.0.0        :  7.31.1 (OK)
jedi >=0.17.2;<0.19.0         :  0.18.1 (OK)
jellyfish >=0.7               :  0.9.0 (OK)
jsonschema >=3.2.0            :  4.4.0 (OK)
keyring >=17.0.0              :  23.4.0 (OK)
nbconvert >=4.0               :  6.4.4 (OK)
numpydoc >=0.6.0              :  1.2 (OK)
paramiko >=2.4.0              :  2.8.1 (OK)
parso >=0.7.0;<0.9.0          :  0.8.3 (OK)
pexpect >=4.4.0               :  4.8.0 (OK)
pickleshare >=0.4             :  0.7.5 (OK)
psutil >=5.3                  :  5.8.0 (OK)
pygments >=2.0                :  2.11.2 (OK)
pylint >=2.5.0                :  2.9.6 (OK)
pyls_spyder >=0.4.0           :  0.4.0 (OK)
pylsp >=1.3.2;<1.4.0          :  1.3.3 (OK)
pylsp_black >=1.0.0           :  1.0.0 (OK)
qdarkstyle =3.0.2             :  3.0.2 (OK)
qstylizer >=0.1.10            :  0.1.10 (OK)
qtawesome >=1.0.2             :  1.0.3 (OK)
qtconsole >=5.2.1;<5.3.0      :  5.2.2 (OK)
qtpy >=1.5.0                  :  2.0.1 (OK)
rtree >=0.9.7                 :  0.9.7 (OK)
setuptools >=49.6.0           :  61.2.0 (OK)
sphinx >=0.6.6                :  4.4.0 (OK)
spyder_kernels >=2.2.1;<2.3.0 :  2.2.1 (OK)
textdistance >=4.2.0          :  4.2.1 (OK)
three_merge >=0.1.1           :  0.1.1 (OK)
watchdog >=0.10.3             :  2.1.6 (OK)
zmq >=17                      :  22.3.0 (OK)

# Optional:
cython >=0.21                 :  0.29.28 (OK)
matplotlib >=2.0.0            :  3.5.1 (OK)
numpy >=1.7                   :  1.21.5 (OK)
pandas >=1.1.1                :  1.4.2 (OK)
scipy >=0.17.0                :  1.7.3 (OK)
sympy >=0.7.3                 :  1.10.1 (OK)
```

**Comment by ccordoba12** [comment]
Hey @walterty, thanks for reporting. This error is already fixed in our latest version (5.3.2).

Since it's still not available in Anaconda, you have two options:

1. Use our Windows installer, which you can find [here](https://github.com/spyder-ide/spyder/releases).
2. Create a new environment with conda-forge packages. For that, please close Spyder, open the Anaconda Prompt and run the following commands there:

        conda create -n spyder-cf -c conda-forge spyder
        conda activate spyder-cf
        spyder

**Comment by walterty** [comment]
Thank you @ccordoba12!! Appreciate the help and quick response.
